businessRules/donations.js

```diff
@@ -21,34 +25,26 @@ function createDonation(firstName, sum, userId, ideaId) {
 }
 
 function ideasDonationSum(ideaId) {
-    const ideas = getAllIdeas();
-    const idea = ideas.find((idea) => idea.id === ideaId);
+    let idea = checkIfIdeaIdMatchesIdea(ideaId);
```

Comment:
The `checkIfIdeaIdMatchesIdea` name suggests that this function returns a boolean value, but surprisingly, it returns an idea. The name of this function should suggest what the function returns

Reply:
I think I've thought of something ) Commited

Reply:
I don't see any new changes

Reply:
That's odd.
![Screenshot_7](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/1c7e44d5-4a1f-4390-bce4-749c7f1c7e15)

Reply:
My bad, It was commited to FR-77, just merged it.